folding/base/miner.py

```diff
@@ -60,6 +60,8 @@ def __init__(self, config=None):
             forward_fn=self.forward,
             blacklist_fn=self.blacklist,
             priority_fn=self.priority,
+        ).attach(
+            forward_fn=self.job_submission_forward, #not sure if we need blacklist on this.
```

Comment:
You attach `job_submission_forward` but not the new function you made called `ping_forward`